workspaces/multi-source-security-viewer/backstage.json

```diff
@@ -0,0 +1 @@
+{ "version": "1.35.0" }
```

Comment:
Can we version bump to the latest, please?

Reply:
Bumped to latest.